src/coreclr/vm/jitinterface.cpp

```diff
@@ -6382,7 +6412,7 @@ DWORD CEEInfo::getMethodAttribsInternal (CORINFO_METHOD_HANDLE ftn)
     if (pMD->IsFCallOrIntrinsic())
         result |= CORINFO_FLG_NOGCCHECK | CORINFO_FLG_INTRINSIC;
     if (pMD->IsJitIntrinsic())
```

Comment:
```suggestion
    if (pMD->IsFCall())
        result |= CORINFO_FLG_NOGCCHECK;
    if (pMD->IsIntrinsic() || pMD->IsArray())
```
Also, you can rename `IsJitIntrinsic` to just `IsIntrinsic` and delete `IsFCallOrIntrinsic` method (the other use of `IsFCallOrIntrinsic` can be replaced with `IsFCall`)